challenge.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# rubocop:disable all
+
+def decode_char(char)
+  morse_dict = { 'a' => '.-', 'b' => '-...', 'c' => '-.-.', 'd' => '-..', 'e' => '.', 'f' => '..-.',
+                 'g' => '--.', 'h' => '....', 'i' => '..', 'j' => '.---', 'k' => '-.-', 'l' => '.-..', 'm' => '--',
+                 'n' => '-.', 'o' => '---', 'p' => '.--.', 'q' => '--.-', 'r' => '.-.', 's' => '...', 't' => '-',
+                 'u' => '..-', 'v' => '...-', 'w' => '.--', 'x' => '-..-', 'y' => '-.--', 'z' => '--..', ' ' => ' ' }
+  letter = ''
+  morse_dict.each { |key, value| if char == value letter = key.upcase
```

Comment:
Great job @c3c1l1a 🤝

- [ ] Kindly check your code on line 10. There seems to be a syntax error.
![error](https://user-images.githubusercontent.com/63917725/187705527-562db9ad-74d2-4d0e-89e5-0b644df25eb7.PNG)